.github/workflows/test.yaml

```diff
@@ -9,17 +9,22 @@ on:
       - '**/*'
   workflow_call:
 
+env:
+  ORG_GRADLE_PROJECT_ossrhUsername: ${{ secrets.OSSRH_USERNAME }}
+  ORG_GRADLE_PROJECT_ossrhPassword: ${{ secrets.OSSRH_PASSWORD }}
+  CI: true
+
 jobs:
   test-android-sdk:
     runs-on: ubuntu-latest
     steps:
       - name: Check out Java SDK
         uses: actions/checkout@v4
 
-      - name: Set up JDK 11
+      - name: Set up JDK 17
```

Comment:
My understanding was we need to use Java 11 for Android. Has this changed? 🤞

Reply:
IIRC Android Gradle Plugin v8 and above require java 17 to run